spec/integration/runner/variant-reassignment-example-14.spec.js

```diff
@@ -0,0 +1,91 @@
+ |         |                            | id: 1                      | id: 2                      |                    | id: 1                      |                                                             |
+ |         |                            | slug: { en: "product" }    | slug: { en: "product-2" }  |                    | slug: { en: "product" }    |                                                             |
+ |         |                            | product-type: "pt2"        | product-type: "pt2"        |                    | product-type: "pt2"        |                                                             |
+ |         |                            | masterVariant: { sku: v1 } | masterVariant: { sku: v2 } |                    | masterVariant: { sku: v1 } |                                                             |
+ |         |                            | variants: { sku: v3 }      |                            |                    |                            |                                                             |
+ +---------+----------------------------+----------------------------+----------------------------+--------------------+----------------------------+-------------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  const logger = utils.createLogger(__filename)
+  let ctpClient
+  let product1
+  let product2
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+
+    let product1Response = await utils.createCtpProducts([['1', '3']], ctpClient)
+    product1 = product1Response[0]
+    product1Response = await ctpClient.products
+      .byId(product1.id)
+      .update({
+        version: product1.version,
+        actions: [{ action: 'publish' }]
+      })
+    await ctpClient.products
+      .byId(product1.id)
+      .update({
+        version: product1Response.body.version,
+        actions: [
+          { action: 'removeVariant', id: product1.variants[0].id }
+        ]
+      })
+
+    const product2Response = await utils.createCtpProducts([['2']], ctpClient)
+    product2 = product2Response[0]
+    await ctpClient.products
+      .byId(product2.id)
+      .update({
+        version: product2.version,
+        actions: [{ action: 'publish' }]
+      })
+  })
+
+  after(async () => {
+    await utils.unpublishAllProducts(ctpClient, logger)
+    await utils.deleteResourcesAll(ctpClient, logger)
+  })
+
+  it('different products on staged and current AND different variants on staged and current',
+    async () => {
+      const reassignment = new VariantReassignment([], logger, {})
+      await reassignment.execute([{
+        productType: product1.productType,
+        name: product1.name,
+        slug: product1.slug,
+        masterVariant: product1.masterVariant,
+        variants: [{}]
```

Comment:
Here the productDraft does not have any variants but in the documentation above and in the `except()` bellow we are expecting variant v2.

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/19/commits/59c189a49f6261b2e50ca15c8854da4beda4719c